### PR TITLE
refactor: more clint

### PR DESCRIPTION
**Pull request by lewis6991**
I know we say the regexp files are off limits, but almost all the clint errors were about the comment style.

### CODE REVIEW

**Comment by zeertzjq** [comment]
Change trailing backslash in line comment to `{backslash}`

**Comment by lewis6991** [comment]
The CI failure can be ignored.

File: src/nvim/msgpack_rpc/helpers.c [hunk 1]
```diff
@@ -95,8 +95,8 @@ bool msgpack_rpc_to_object(const msgpack_object *const obj, Object *const arg)
       dest = conv(((String) { \
       .size = obj->via.attr.size, \
       .data = (obj->via.attr.ptr == NULL || obj->via.attr.size == 0 \
-                   ? xmemdupz("", 0) \
-                   : xmemdupz(obj->via.attr.ptr, obj->via.attr.size)), \
+               ? xmemdupz("", 0) \
+               : xmemdupz(obj->via.attr.ptr, obj->via.attr.size)), \
     })); \
```
Comment (dundargoc) [inline src/nvim/msgpack_rpc/helpers.c:100]:
```suggestion
               : xmemdupz(obj->via.attr.ptr, obj->via.attr.size)), })); \
```

Comment (dundargoc) [inline src/nvim/msgpack_rpc/helpers.c:100]:
```suggestion
               : xmemdupz(obj->via.attr.ptr, obj->via.attr.size)), })); \
```

**Review by justinmk — APPROVED** [review]
needs rebase

**Pull request merged**